agents/aliyun/fence_aliyun.py

```diff
@@ -121,12 +122,20 @@ def define_new_opts():
 		"required" : "0",
 		"order" : 4
 	}
+	all_opt["ram_role"] = {
+		"getopt": "m:",
```

Comment:
Can you update this to `":"`, and remove `-m` from the help text? I dont think it makes sense having `-m` for the `ram-role`.